examples/amp-story/amp-story-shopping.html

```diff
@@ -17,21 +17,65 @@
         publisher="AMP Story Shopping"
         publisher-logo-src="example.com/logo.png"
         poster-portrait-src="example.com/poster.jpg">
-      <amp-story-page id="default">
-        <amp-story-shopping-config layout='nodisplay'></amp-story-shopping-config>
+      <amp-story-page id="example">
+        <!--
+          This is an example of a JSON config retrieved from a source file.
+          To use the inline JSON config, remove the src attribute.
+        -->
```

Comment:
nit: It's good to keep comments brief, this could be something like:
```
Example of JSON retrieved from src attribute.
Falls back to inline if request fails.
```